schema_utils_test.py

```diff
@@ -269,19 +277,25 @@ def _validate_validator(obj_type, validator):
 
     # Check that the id corresponds to a valid normalizer function.
     validator_fn = schema_utils.get_validator(validator['id'])
-    assert set(inspect.getargspec(validator_fn).args) == set(
+    assert set(inspect.getfullargspec(validator_fn).args) == set(
```

Comment:
Adding a comment here for future reference. The function `getargspec` has now been [deprecated](https://docs.python.org/3/library/inspect.html#inspect.getargspec) since Python version 3.0, and we should now use `getfullargspec` instead. `getargspec` doesn't support parsing type annotations, and so devs might be facing errors (like this - `ValueError: Function has keyword-only parameters or annotations, use getfullargspec() API which can support them`) in future because of that in test files after they type annotate the corresponding code files.

@DubeySandeep would it make sense to change rest of the usage of `getargspec` now itself? (For context, change it back to `getargspec` and run the test file after pulling from this branch, you will see the above mentioned error)